roles/telemetry_autoscaling/tasks/configure_heat.yml

```diff
@@ -27,19 +27,19 @@
 
   # TODO: Pass the parameters correctly. When I tried to pass the parameters
   # into the template.yaml file, they weren't passed to the instance initially.
-- name: RHOSO-12657 Configure heat template for automatically scaling instances
+- name: [TEST] configure heat template automatically scaling instances
   ansible.builtin.template:
     src: instance.yaml.j2
     dest: ~/templates/autoscaling/vnf/instance.yaml
 
-- name: RHOSO-12658 Create the resource to reference in the heat template
+- name: [TEST] create resource reference heat template
```

Comment:
I think removing "the, to , in the" making it a little bit difficult to understand what's meant by this task.
do we have a constrain on the words count ?

Reply:
This is to match the polarian test-automation-ids.
if the task is unclear, it is totally possible to make the task names multiple lines long to add some more information